index.html

```diff
@@ -595,8 +594,8 @@ <h4>TTML Format</h4>
       <p class="issue" data-number="25"></p>
       <section>
         <h4>Identification and Profile Signaling</h4>
-        <p>TTML documents representing <a>DAPT Scripts</a> shall be compliant with the Text Profile defined in [[!ttml-imsc1.1]] with the additional constraints defined in this specification.</p>
-        <p>TTML documents representing <a>DAPT Scripts</a> shall specify a <code>ttp:contentProfiles</code> attribute on the <code>tt</code> element with one value equal to <code>http://www.w3.org/ns/ttml/profile/dapt</code>. Other values such as the text profile designator [[!ttml-imsc1.1]] of MAY be present.</p>
+        <p>TTML documents representing <a>DAPT Scripts</a> MUST be compliant with the Text Profile defined in [[!ttml-imsc1.1]] with the additional constraints defined in this specification.</p>
+        <p>TTML documents representing <a>DAPT Scripts</a> MUST specify a <code>ttp:contentProfiles</code> attribute on the <code>tt</code> element with one value equal to <code>http://www.w3.org/ns/ttml/profile/dapt</code>. Other values such as the text profile designator [[!ttml-imsc1.1]] of MAY be present.</p>
```

Comment:
I thought we'd agreed to drop this section, because it's duplicated elsewhere?

Reply:
Agree to remove

Reply:
Deleted the section and added a comment to https://github.com/w3c/dapt/issues/43